app/decorators/ephemera_folder_decorator.rb

```diff
@@ -214,6 +210,7 @@ def find_resource(resource_id)
       resource_id
     end
 
+    # Unsure if I should move this to wayfinder.
```

Comment:
I think this relationship is a little different from the others.  But I do expect we'll have more use of controlled vocabs in the future, so finding a good pattern for handling these seems good.